### PR TITLE
Include Location And Role Default Values In UG

### PR DESCRIPTION
**Pull request by HM33-Stu**
Make amendments to location and role description in UG to include explanation about default values
Update: Default values explanation removed

Related PR (actual change to code): https://github.com/AY2324S2-CS2103T-W11-1/tp/pull/258

### CODE REVIEW

**Comment by HM33-Stu** [comment]
Amendments Made;

image
image

File: docs/UserGuide.md [hunk 1]
```diff
@@ -372,11 +372,12 @@ Lets you add an internship entry to be tracked by CareerSync!
 
 **Format:** `add /com COMPANY_NAME /desc DESCRIPTION /status STATUS /poc CONTACT_NAME /email CONTACT_EMAIL /phone CONTACT_NUMBER [/loc LOCATION_ENUM] [/role ROLE] ​` <br>
 
-* The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` and `ROLE` allow the use of any text, number and/or spaces
+* The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` allow the use of any text, number and/or spaces
 * The field `STATUS` accepts only the following inputs: `to_apply`, `ongoing`, `rejected`, `accepted` (case-insensitive)
 * The field `CONTACT_EMAIL` accepts only the format of `<email>@<domain>.com`
 * The field `CONTACT_NUMBER` must be unsigned, and it must be at least 7 digits long
-* The field `LOCATION_ENUM` accepts only the following inputs: `remote`, `local`, `overseas` (case-insensitive)
+* The field `LOCATION_ENUM` accepts only the following inputs: `unknown`, `remote`, `local`, `overseas` (case-insensitive)
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:379]: For UG, i think important to specify that you can have it blank to be defaulted to unknown

File: docs/UserGuide.md [hunk 2]
```diff
@@ -372,11 +372,12 @@ Lets you add an internship entry to be tracked by CareerSync!
...
+* The field `ROLE` allow the use of any text, number and/or spaces
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:380]: Maybe an easier description would be any ASCII character EXCEPT '!' and '?', now it doesnt include the fact that you can have symbols.

**Review by CaptainKeqing — DISMISSED** [review]
just small changes

File: docs/UserGuide.md [hunk 3]
```diff
@@ -398,11 +398,13 @@ Lets you add an internship entry to be tracked by CareerSync!
...
 
-* The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` and `ROLE` allow the use of any text, number and/or spaces
+* The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` allow the use of any text, number and/or spaces
 * The field `STATUS` accepts only the following inputs: `to_apply`, `ongoing`, `rejected`, `accepted` (case-insensitive)
 * The field `CONTACT_EMAIL` accepts only the format of `<email>@<domain>`
 * The field `CONTACT_NUMBER` must be unsigned, and it must be at least 7 digits long
-* The field `LOCATION_ENUM` accepts only the following inputs: `remote`, `local`, `overseas` (case-insensitive)
+* The field `LOCATION_ENUM` accepts only the following inputs: `unknown`, `remote`, `local`, `overseas` (case-insensitive)
+  * Do note that should you leave the location field empty, it will default to `unknown`
+* The field `ROLE` allow the use of any ASCII characters (excluding `!` and `?`)
```
Comment (HM33-Stu) [inline docs/UserGuide.md:407]: @CaptainKeqing Requested changes made

**Review by CaptainKeqing — APPROVED** [review]
Looks good to me, changes are made.

**Pull request merged**